daemon/swap_daemon_erc20_test.go

```diff
@@ -29,15 +30,17 @@ func TestRunSwapDaemon_ExchangesXMRForERC20Tokens(t *testing.T) {
 
 	aliceConf := CreateTestConf(t, tests.GetTakerTestKey(t))
 
-	tokenAddr := GetMockTokens(t, aliceConf.EthereumClient)[MockTether]
-	tokenAsset := types.EthAsset(tokenAddr)
+	tokenAsset := getMockTetherAsset(t, aliceConf.EthereumClient)
 
 	timeout := 7 * time.Minute
 	ctx, _ := LaunchDaemons(t, timeout, aliceConf, bobConf)
 
 	bc := rpcclient.NewClient(ctx, bobConf.RPCPort)
 	ac := rpcclient.NewClient(ctx, aliceConf.RPCPort)
 
+	bobStartTokenBal, err := bobConf.EthereumClient.ERC20Balance(ctx, tokenAsset.Address())
+	require.NoError(t, err)
```

Comment:
Now that we have more token tests and are caching the token contract between tests, we can't assume that Bob's token balance is zero on start.